### DISCUSSION TITLE
Dark Mode Issue | Random class changes

### DISCUSSION THREAD

**Discussion by orilpik** [tailpress/tailpress#207] [General]
Hi!

I'm having some trouble with Tailpress: I use dark: classes in some points of my code but they only works in the my menu container, not on the rest of the code, even the main body. Can anyone help me to find out the problem? I'd tried to manually put "darkMode":"class" on the js of the plugin, but then my custom classes of the options editor didn't work.

My dev website to check: https://dev01.urquiabas.com/ (You'll find the only div that dark mode work in <header></header>

**Comment by jeffreyvr** [comment]
Hi,

It's difficult to debug because I'm not sure of the setup. In general, it could be because:

- Some files are maybe not being watched (depends on your setup) https://github.com/jeffreyvr/tailpress/blob/master/tailwind.config.js#L6
- Caching 

When I add dark mode, I follow the way it's outlined above on this page:
https://tailwindcss.com/docs/dark-mode

Then when all files are being watched, it should work.

**Reply by orilpik** [reply]
Thanks for the quick answer! I find out that it works on the footer.php, header.php and the base files of the theme, but not in the ones that I created (that are custom page php files that work as page templates) I didn't touch the configuration on watching, is there anything else I have to do?
